packages/web-components/@sfx/sayt/stories/index.ts

```diff
@@ -1,25 +1,48 @@
 import { storiesOf } from '@storybook/html';
 import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+
```

Comment:
remove extra space ?

Reply:
I left this here on purpose to separate 3rd party imports from local ones, but I can do whatever the team prefers.

Reply:
I'm in favour of not having spaces because you can clearly see which ones are local and which ones are not based on the path, but I haven't been flagging them since we haven't had the discussion yet.